src/components/previewers/ImagePreviewer.jsx

```
@@ -0,0 +1,150 @@
+import React, { useCallback, useMemo, useState } from "react";
+import { IoHeart, IoChatbubbleEllipses, IoSend } from "react-icons/io5";
+import { FaCopy } from "react-icons/fa";
+import ProfileCard from "../card/ProfileCard";
+import Modal from "../modal/Modal";
+import { useSelector } from "react-redux";
+import AppInput from "../form/AppInput";
+
+const ImagePreviewer = ({
+  image,
```

ESLint

[error] 'image' is missing in props validation

(react/prop-types)